spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require 'byebug'
```

Comment:
Pelo que entendi, você deixou o `byebug` configurado no ambiente de testes. Quando quisermos usar, basta inserir `byebug` em qualquer caso de testes para ter acesso ao debug. Se for isso, acha que cabe aqui um comentário explicando? Ou talvez no `README.md`. Só pra evitar que a gente remova esse `require` por achar que não está sendo usado em lugar nenhum.

Reply:
@ingoncalves

Reply:
@ingoncalves

Reply:
Boa! Feito em 38342adf152239ac0eb806ee953d11dbc13f1f83